src/Sylius/Bundle/AdminBundle/Resources/config/app/sylius/resources/admin_user.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use Sylius\Bundle\AdminBundle\Form\Type\AdminUserType;
+use Sylius\Resource\Metadata\BulkDelete;
+use Sylius\Resource\Metadata\Create;
+use Sylius\Resource\Metadata\Delete;
+use Sylius\Resource\Metadata\Index;
+use Sylius\Resource\Metadata\Operations;
+use Sylius\Resource\Metadata\ResourceMetadata;
+use Sylius\Resource\Metadata\Update;
+
+return (new ResourceMetadata())
+    ->withRoutePrefix('/admin')
+    ->withClass('%sylius.model.admin_user.class%')
+    ->withSection('admin')
+    ->withTemplatesDir('@SyliusAdmin\\shared\\crud')
```

Comment:
I see this a lot but Symfony can handle both / and \ format.

To avoid using too many chars this can be simplified like this:
```suggestion
    ->withTemplatesDir('@SyliusAdmin/shared/crud')
```

Same for the rest of the resource definitions

Reply:
Yes, I agree.